data/tribes/workers/frisians/charcoal_burner/init.lua

```diff
@@ -29,6 +29,10 @@ tribes:new_worker_type {
          "callobject=with_stack",
          "animate=stacking_2 1000",
          "return"
+      },
+      find_pond = {
```

Comment:
Do older savegames still load?

Reply:
they should load in any case as adding a new program never broke anything. Only deleting and renaming is tricky as the old status is saved in the savegame. By the way this is the same mechanism as in the last turn withthe 3 Programs performance turn. There we finally added a second program as well. But compatibility only broke cause I renamed the old program which I then reverted.   
I have no savegame to test this though.